agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -286,6 +286,9 @@ void backhaul_manager::platform_notify_error(int code, const std::string &error_
     message_com::send_cmdu(m_scPlatform.get(), cmdu_tx);
 }
 
+void backhaul_manager::before_select()
+{
```

Comment:
How can this compile??? you're missing a `}`

Reply:
It was added on later commit. Guess I forgot to git add the close bracket on that specific commit and added it on one of the following commits. Anyway, fixed.